packaging/systemd/configure-ovs-microshift.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -eux
+
+OVS_HANDLER_THREADS=${OVS_HANDLER_THREADS:-1}
+OVS_REVALIDATOR_THREADS=${OVS_REVALIDATOR_THREADS:-1}
+
+# let ovn-controller know we are using an unix socket for SBDB communication instead of TLS/TCP
+ovs-vsctl set open . external-ids:ovn-remote="unix:/var/run/ovn/ovnsb_db.sock"
+
+# limit the number of threads in ovs-vswitchd to limit the memory consumption
+
+if [[ "${OVS_HANDLER_THREADS}" != "auto" ]]; then
+    ovs-vsctl set open . other_config:n-handler-threads=${OVS_HANDLER_THREADS}
```

Comment:
This won't be applied with openvswitch2.16 version.

Reply:
I saw handling it on the logs, it's fine though even if it's not fully effective, will be when we switch to 2.17

Reply:
I think you mean, it won't be applied with 2.17?

Reply:
I wonder if we can leave it in place (if harmless) and have the ovs team re-admit that parameter so we don't need to touch the cpu affinity?